source/common/tracing/http_tracer_impl.cc

```diff
@@ -17,21 +17,6 @@
 namespace Envoy {
 namespace Tracing {
 
-static std::string buildRequestLine(const Http::HeaderMap& request_headers,
-                                    const Http::AccessLog::RequestInfo& info) {
-  std::string path = request_headers.EnvoyOriginalPath()
-                         ? request_headers.EnvoyOriginalPath()->value().c_str()
-                         : request_headers.Path()->value().c_str();
-  static const size_t max_path_length = 128;
-
-  if (path.length() > max_path_length) {
-    path = path.substr(0, max_path_length);
-  }
-
-  return fmt::format("{} {} {}", request_headers.Method()->value().c_str(), path,
-                     Http::AccessLog::AccessLogFormatUtils::protocolToString(info.protocol()));
```

Comment:
We are losing protocol with this change. Do we care about this? Should we put this into a new tag?